src/__tests__/parser.test.ts

```diff
@@ -148,19 +149,19 @@ describe("Parsing CSVs ", () => {
     expect(paymentWithoutDecimal.decimals).to.be.equal(18);
     expect(paymentWithoutDecimal.receiver).to.equal(validReceiverAddress);
     expect(paymentWithoutDecimal.tokenAddress).to.equal(listedToken.address);
-    expect(paymentWithoutDecimal.amount.isEqualTo(new BigNumber(1))).to.be.true;
+    expect(paymentWithoutDecimal.value.isEqualTo(new BigNumber(1))).to.be.true;
     expect(paymentWithoutDecimal.receiverEnsName).to.be.null;
 
     expect(paymentWithDecimal.receiver).to.equal(validReceiverAddress);
     expect(paymentWithDecimal.tokenAddress?.toLowerCase()).to.equal(listedToken.address.toLowerCase());
     expect(paymentWithDecimal.decimals).to.equal(18);
-    expect(paymentWithDecimal.amount.isEqualTo(new BigNumber(69.42))).to.be.true;
+    expect(paymentWithDecimal.value.isEqualTo(new BigNumber(69.42))).to.be.true;
     expect(paymentWithDecimal.receiverEnsName).to.be.null;
 
     expect(paymentWithoutTokenAddress.decimals).to.be.equal(18);
     expect(paymentWithoutTokenAddress.receiver).to.equal(validReceiverAddress);
     expect(paymentWithoutTokenAddress.tokenAddress).to.equal(null);
-    expect(paymentWithoutTokenAddress.amount.isEqualTo(new BigNumber(1))).to.be.true;
+    expect(paymentWithoutTokenAddress.value.isEqualTo(new BigNumber(1))).to.be.true;
```

Comment:
This PR would have way fewer changes if we kept everything as "amount" and only allow value to be used in place of amount.